src/components/view/ListView.vue

```diff
@@ -537,4 +593,16 @@ export default {
     }
 
   }
+
+  .alert-notification-threshold {
+    background-color: rgba(255, 231, 175, 0.75);
+    color: #e87900;
+    padding: 10%;
```

Comment:
@davidjumani instead of percentage should we say 10px?

Reply:
px works fine for a larger screen but percentage works better when with mobile!